oshi-core/src/main/java/oshi/driver/windows/wmi/Win32Process.java

```diff
@@ -72,7 +72,7 @@ public static WmiResult<CommandLineProperty> queryCommandLines(Set<Integer> pids
         String sb = WIN32_PROCESS;
         if (pidsToQuery != null) {
             sb += " WHERE ProcessID="
-                    + pidsToQuery.stream().map(String::valueOf).collect(Collectors.joining(" OR PROCESSID="));
+                    + pidsToQuery.stream().parallel().map(String::valueOf).collect(Collectors.joining(" OR PROCESSID="));
```

Comment:
Any performance benefit here is dwarfed by the resulting WMI call overhead.  Probably doesn't hurt, but I doubt it's a noticeable help.